thentos-core/devel.config

```diff
@@ -1,3 +1,7 @@
+# This "secret" is hardcoded and public.
+# If you remove it, thentos will generate a new one for you.
+csrf_secret: 1daf3741e8a9ae1b39fd7e9cc7bab44ee31b6c3119ab5c3b05ac33cbb543289c
```

Comment:
if this can be generated implicitly, then what's the reason it is in the config?  so that form submission works in between server restarts?